mtp_cashbook/apps/disbursements/views.py

```diff
@@ -437,8 +445,8 @@ def create_nomis_transaction(self, form):
                 record_id='d%s' % disbursement['id'],
                 description='Sent to {recipient_first_name} {recipient_last_name}'.format(
                     recipient_first_name=disbursement['recipient_first_name'],
-                    recipient_last_name=disbursement['recipient_last_name']
-                ),
+                    recipient_last_name=disbursement['recipient_last_name'],
+                ).replace('  ', ' '),
```

Comment:
Is this replacing double spaces with single spaces? I would think it would be better to trim the inputs. Or just don't worry about it, why are people putting spaces at the start/end of names anyway?